engines/bladerunner/bladerunner.h

```diff
@@ -351,10 +351,10 @@ static inline void drawPixel(Graphics::Surface &surface, void* dst, uint32 value
 			*(uint8*)dst = (uint8)value;
 			break;
 		case 2:
-			*(uint16*)dst = (uint16)value;
+			WRITE_UINT16(dst, (uint16)value);
```

Comment:
Are these really needed? Dst should be always aligned properly for each pixel and with selected cast for write. After all it is read subsequently by cpu or gpu.
With a profiler I can see that this is causing a bigger regression ~4% of whole runtime

Reply:
on the other hand, it is not visible in a release build, but sadly all of our nightlies have disabled optimizations :/

Reply:
It looks like it's not needed after all. I think I was just being overly cautious.